utils/ccdatofhir/mapping_datatypes.bal

```diff
@@ -0,0 +1,221 @@
+    xml postalCodeElement = addressElement/<v3:postalCode|postalCode>;
+
+    string streetAddressLine = streetAddressLineElement.data().trim();
+    string city = cityElement.data().trim();
+    string state = stateElement.data().trim();
+    string postalCode = postalCodeElement.data().trim();
+
+    if streetAddressLine != "" || city != "" || state != "" || postalCode != "" {
+        return {
+            line: [streetAddressLine],
+            city: city,
+            state: state,
+            postalCode: postalCode
+        };
+    } else {
+        log:printDebug("Address fields not available");
+        return ();
+    }
+}
+
+# Map C-CDA telecom to FHIR ContactPoint.
+#
+# + telecomElement - C-CDA telecom element
+# + return - Return FHIR ContactPoint
+public isolated function mapCcdaTelecomToFhirTelecom(xml telecomElement) returns r4:ContactPoint? {
+    string|error? telecomUse = telecomElement.use;
+    string|error? telecomValue = telecomElement.value;
+
+    string? valueVal = ();
+    if telecomValue is string {
+        valueVal = telecomValue;
+    } else {
+        log:printDebug("Telecom value not available", telecomValue);
+    }
+
+    r4:ContactPointUse? useVal = ();
+    if telecomUse is string {
+        match telecomUse {
+            "HP" => {
+                useVal = "home";
+            }
+            "WP" => {
+                useVal = "work";
+            }
+        }
+    } else {
+        log:printDebug("Telecom use not available", telecomUse);
+    }
+
+    if valueVal is string && useVal is string {
+        return {
+            value: valueVal,
+            use: useVal
+        };
+    } else {
+        log:printDebug("telecom fields not available");
+        return ();
+    }
+}
+
+# Map C-CDA name to FHIR HumanName.
+#
+# + nameElement - C-CDA name element
+# + return - Return FHIR HumanName
+public isolated function mapCcdaNametoFhirName(xml nameElement) returns r4:HumanName? {
+    xml familyElement = nameElement/<v3:family|family>;
+    xml givenElement = nameElement/<v3:given|given>;
+
+    string givenName = givenElement.data().trim();
+    string family = familyElement.data().trim();
+
+    if givenName != "" || family != "" {
+        return {given: [givenElement.data()], family: familyElement.data()};
+    } else {
+        log:printDebug("name fields not available");
+        return ();
+    }
+}
+
+# Map C-CDA code to FHIR CodeableConcept.
+#
+# + codingElement - C-CDA code element
+# + return - Return FHIR CodeableConcept
+public isolated function mapCcdaCodingtoFhirCodeableConcept(xml codingElement) returns r4:CodeableConcept? {
+    r4:Coding? mapCcdaCodingtoFhirCodeResult = mapCcdaCodingtoFhirCode(codingElement);
+    if mapCcdaCodingtoFhirCodeResult is r4:Coding {
+        return {
+            coding: [mapCcdaCodingtoFhirCodeResult]
+        };
+    } else {
+        log:printDebug("codeableConcept not available");
+        return ();
+    }
```

Comment:
```suggestion
    } 
    log:printDebug("codeableConcept not available");
```

Reply:
Can do the same with other similar cases as well

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/a482466dc4ba847b3e57e8e27d7d0e8cc48cf07c